### PR TITLE
Add test for Ruler class

### PR DESCRIPTION
**Pull request by joaofoes**

### CODE REVIEW

File: ADR/Checkers/tests/test_Dimensions.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from ADR.Components.Plane import Plane
+from ADR.Checkers.Dimensions import Ruler
+from ADR.parameters import get_plane_parameters, get_performance_parameters
```
Comment (rafaellehmkuhl) [inline ADR/Checkers/tests/test_Dimensions.py:3]: get_performance_parameters is imported but not used

You can remove it.

File: ADR/Checkers/tests/test_Dimensions.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import numpy.testing as npt
+import pytest
```
Comment (rafaellehmkuhl) [inline ADR/Checkers/tests/test_Dimensions.py:5]: pytest is also imported but not used, so it's ok to remove it too.

File: ADR/Checkers/tests/test_Dimensions.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    plane_dimensions = Ruler(plane)
+    plane_dimensions.measure()
+
+    dimensions_check = 3.602
+
+    npt.assert_almost_equal(
+        plane_dimensions.plane.total_dimensions, dimensions_check, decimal=3
+    )
+    npt.assert_equal(plane_dimensions.plane.dimensions_are_good, True)
```
Comment (rafaellehmkuhl) [inline ADR/Checkers/tests/test_Dimensions.py:20]: Perfect.

**Comment by rafaellehmkuhl** [comment]
Fixes #146

**Comment by joaofoes** [comment]
Same problem as the Power class PR!

**Pull request merged**